package-lock.json

```diff
@@ -1204,6 +1204,11 @@
             }
           }
         },
+        "tslib": {
```

Comment:
Wondering howcome there is a change here? Don't think anything was changed in package.json 🤔